dotnet/Razorvine.Pyrolite/Benchmarks/PicklerBenchmarks.cs

```diff
@@ -0,0 +1,69 @@
+﻿using BenchmarkDotNet.Attributes;
+using Razorvine.Pickle;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Benchmarks
+{
+    [GenericTypeArguments(typeof(string))]
+    [GenericTypeArguments(typeof(double))]
+    [GenericTypeArguments(typeof(int))]
+    [GenericTypeArguments(typeof(bool))]
+    public class PicklerBenchmarks<T>
+    {
+        [Params(1000)]
+        public int Count { get; set; }
+
+        [Params(false)]
+        public bool UseMemo { get; set; }
```

Comment:
The default Pickler constructor uses `useMemo: true`, but we aren't benchmarking that. Is there a reason?

Reply:
False is the default in dotnet/spark. But you are right, we should measure both cases. I added the `useMemo: true` case.